packages/protocol/test/thirdparty/LibFixedPointMath.t.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
@kirk-baird Do you mind adding this license to all files that you add and maybe also add 
```
/// @author Your Name <your@email.address>
```

Reply:
Yep done, I believe I've got all of the files now.

I'm not a lawyer but I believe since we were contracted by you, you've got ownership of the tests IP so should be able to use your license if you'd like?

Reply:
Using our license works for us. Thank you.

Reply:
I'll update the license myself later:)